src/screens/Dashboard/index.js

```diff
@@ -218,32 +215,29 @@ export default class Home extends React.Component {
                 settings={settings}
                 configuration={configuration}
                 settingsList={settingsList}
-                configurationList={configurationList}
-                onSave={({
-                  settings: savedSettings,
-                  configuration: savedConfiguration,
-                }) => {
-                  let configList = configurationList;
-                  let config = savedConfiguration;
-
+                onSave={({ config, settings: savedSettings }) => {
                   if (
                     settings.ocppConfiguration !==
                     savedSettings.ocppConfiguration
                   ) {
-                    configList = getConfigurationList(
+                    const newConfiguration = getConfiguration(
                       savedSettings.ocppConfiguration
                     );
-                    config = getConfiguration(savedSettings.ocppConfiguration);
-                  }
 
+                    this.setState({
+                      configuration: newConfiguration,
+                    });
+                    chargeStation.changeConfiguration(newConfiguration);
+                  } else {
+                    configuration.updateVariablesFromSimpleConfigurationMap(
+                      config
+                    );
+                  }
```

Comment:
This still requires some improvement. Right now you first have to change the OCPP version, then 'save', then go back to the settings and change all the other configurations as we don't port 1.6 variables to 2.0.1 and vice versa.

Reply:
Will think about this a bit more.